tests/handlers/test_presence.py

```diff
@@ -716,19 +718,41 @@ def test_set_presence_from_syncing_keeps_status(self):
         # our status message should be the same as it was before
         self.assertEqual(state.status_msg, status_msg)
 
-    def test_set_presence_from_syncing_keeps_busy(self):
+    @parameterized.expand([(False,), (True,)])
+    @unittest.override_config(
+        {
+            "experimental_features": {
+                "msc3026_enabled": True,
+            },
+        }
+    )
+    def test_set_presence_from_syncing_keeps_busy(self, test_with_workers: bool):
         """Test that presence set by syncing doesn't affect busy status"""
-        # while this isn't the default
-        self.presence_handler._busy_presence_enabled = True
-
         user_id = "@test:server"
         status_msg = "I'm busy!"
 
+        worker_hs = None
```

Comment:
Can't we default this to `self.hs` to avoid the `if`/`else` blocks further down?

Reply:
Indeed we can. Nice cleanup! Done in dedc990d2115c4cd93aea0493536782071ff58b4.